actions/utils/generate_unique_suffix/action.yml

```diff
@@ -0,0 +1,14 @@
+name: "Generate Image Tag Suffix"
```

Comment:
I'm not sure I follow - there's not a deterministic tag we can apply here? We generate a completely new image tag every time the action runs even if the artifacts it creates are identical?